addons/mail/controllers/thread.py

```diff
@@ -73,6 +73,23 @@ def mail_thread_messages(self, thread_model, thread_id, fetch_params=None):
             "messages": messages.ids,
         }
 
+    @http.route("/mail/thread/recipients", methods=["POST"], type="jsonrpc", auth="user")
+    def mail_thread_recipients(self, thread_model, thread_id, message_id=None, **kwargs):
+        thread = self._get_thread_with_access(thread_model, thread_id, mode='read', **kwargs)
+        if message_id:
+            message = self._get_message_with_access(message_id, mode="create", **kwargs)
```

Comment:
Supporting `kwargs` here makes little sense if the function is limited to internal users (this is only used for portal stuff in other methods).

But how about project chatter from portal users? Is it taken into account at all in all the new tasks about composer? 
Portal should probably have specific route/overrides to turn possible recipients (limited to current followers of task/project)?

Reply:
Yes, had the same thoughts when adding the route. Still wondering if we want that feature for portal or not. Open question would say, makes sense in some cases (e.g. on a ticket, a portal user wants to add their boss, or pings their accounting person on a sale order), maybe not in other (i.e. don't spam partner table). Currently on hold.